content/docs/setup-and-administration/configuring-app-registrations-for-non-interactive-use/index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Configuring App Registrations for Non-Interactive Use"
+description: "Creating App Registrations for RKVST"
+lead: "Creating App Registrations for RKVST"
+date: 2021-06-16T11:12:25+01:00
+lastmod: 2021-06-16T11:12:25+01:00
+draft: false
+images: []
+menu: 
+  docs:
+    parent: "setup-and-administration"
+weight: 12
+toc: true
+---
+
+Non-interactive access to the RKVST platform is managed by creating `App Registrations` either via the Manage RKVST Interface in the UI or by using the App Registrations API directly.
+
+`App Registrations` are Authentication Profiles with a `CLIENT ID` and `SECRET` that can then be used to generate JWT Tokens for authenticating to RKVST API Endpoints.
+
+## Creating an App Registration
+
+To enable non-interactive access to RKVST you **must** create your first `App Registration` in the RKVST UI and **only** as a Root User.
+
+### Using the RKVST UI (Required for First-Time Setup)
+
+1. As a Root User visit the APP REGISTRATIONS tab in the `Manage RKVST` Interface
+2. Click CREATE APP REGISTRATION.
+3. Enter any display name you like.
+a. Optional - using the ADD CUSTOM CLAIM button Add any extra claims you require in your access token.
+4. Click CREATE APP REGISTRATION. The response will include the CLIENTID and SECRET required by the archivist token endpoint.
+
+{{< caution >}}
+**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later.
+{{< /caution >}}
+
+5. Now you have created your App Registration, follow the steps further below to [test generating a token](./#getting-a-token-with-your-app-registration) and [ensure you can access the RKVST API](./#testing-your-access).
+
+### Using the App Registrations API
```

Comment:
Important: This can only happen if the Application is a root user. Do we have guidance on how to set this up?

Reply:
I will add that as part of the additions on Root Users in general

Reply:
I have sort of fixed this by pointing people at the tenancies API in a callout but not expressively; I am wondering if it may be better to add another docs section called 'Managing Users' or something to the Setup and Administration section

That may be neater than stuffing this section with everything

Reply:
Would welcome opinion

Reply:
This is an important and very useful piece of functionality that is quite opaque so I think we should add it.

Reply:
Note that in our current docs it suggest that everybody's issuer is the same so as a root user you can copy your own issuer into the new import box.  THAT DOESN'T WORK ANYMORE! The issuer for Application Registration users is "https://app.rkvst.io/appidpv1".   We need to document that!